model-optimizer/extensions/analysis/inputs.py

```diff
@@ -56,19 +56,21 @@ def ignore_mxnet_softmax_inputs(cls, graph: Graph):
     def iterator_get_next_analysis(cls, graph: Graph, inputs_desc: dict):
         message = None
         op_nodes = graph.get_op_nodes(op='IteratorGetNext')
+
+        params = ''
+        for iter_get_next in op_nodes:
+            for port in iter_get_next.out_nodes().keys():
+                inputs_desc['{}:{}'.format(iter_get_next.soft_get('name', iter_get_next.id), port)] = {
+                    'shape': iter_get_next.shapes[port].tolist(),
+                    'value': None,
+                    'data_type': iter_get_next.types[port]
+                }
+                if params != '':
+                    params = params + ','
+                shape = str(iter_get_next.shapes[port].tolist()).replace(',', '')
+                params = params + '{}:{}{}'.format(iter_get_next.soft_get('name', iter_get_next.id), port, shape)
+
         if len(op_nodes):
-            params = ''
-            for iter_get_next in op_nodes:
-                for port in iter_get_next.out_nodes().keys():
-                    inputs_desc['{}:{}'.format(iter_get_next.soft_get('name', iter_get_next.id), port)] = {
-                        'shape': iter_get_next.shapes[port].tolist(),
-                        'value': None,
-                        'data_type': iter_get_next.types[port]
-                    }
-                    if params != '':
-                        params = params + ','
-                    shape = str(iter_get_next.shapes[port].tolist()).replace(',', '')
-                    params = params + '{}:{}{}'.format(iter_get_next.soft_get('name', iter_get_next.id), port, shape)
             message = 'It looks like there is IteratorGetNext as input\n' \
                       'Run the Model Optimizer with:\n\t\t--input "{}"\n' \
                       'And replace all negative values with positive values'.format(params)
```

Comment:
last comment. please extend a message.
1. "And replace all negative values with positive values" is not necessary now because we support dynamic dimensions
2. we need to mention one more way that is OOB. 

Propose:
'It looks like there is IteratorGetNext as input\n' \
'Run the Model Optimizer without --input option \n' \
'Otherwise, try to run the Model Optimizer with:\n\t\t--input "{}"\n'

Reply:
done